src/main/java/Task.java

```diff
@@ -0,0 +1,36 @@
+import java.time.LocalDate;
+
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "X" : " "); // mark done task with X
+    }
+
+    public void markAsDone() {
+        this.isDone = true;
+    }
+
+    public String getDescription() {
+        return this.description;
+    }
+
+    public String getTask() {
```

Comment:
Would it be more accurate to have named this `getTaskTypeIcon` following the style from `getStatusIcon` in line 12?